src/Client/Module/Modules/Module.cpp

```diff
@@ -9,8 +9,8 @@ std::map<std::string, DWRITE_TEXT_ALIGNMENT> alignments = {
 };
 
 void Module::NormalRender(int index, std::string text, std::string value) {
-
-	if (settings.getSettingByName<bool>("enabled")->value && SDK::CurrentScreen == "hud_screen") {
+	FlarialGUI::inMenu = ModuleManager::getModule("ClickGUI")->settings.getSettingByName<bool>("enabled")->value;
+	if (FlarialGUI::inMenu && SDK::CurrentScreen == "hud_screen") {
```

Comment:
This is an oopsie (FlarialGUI::inMenu instead of settings.getSettingByName("enabled")->value) but I changed it in other commit so it works